marimo/_cli/sandbox.py

```diff
@@ -163,6 +188,9 @@ def construct_uv_flags(
         "--no-project",
         # trade installation time for faster start time
         "--compile-bytecode",
+        # layer additional deps on top of the requirements
+        "--with",
+        ",".join(additional_deps),
```

Comment:
possibly double check if `additional_deps` is empty?

Reply:
Good idea.